cloud-gateway-service/src/main/java/org/zowe/apiml/cloudgatewayservice/service/GatewayIndexService.java

```diff
@@ -58,40 +47,23 @@ public class GatewayIndexService {
     private final Cache<String, ServiceInstance> apimlGatewayLookup;
     private final Cache<String, List<ServiceInfo>> apimlServicesCache;
     private final WebClient defaultWebClient;
-    private SslContext customClientSslContext = null;
 
-    public GatewayIndexService(WebClient defaultWebClient,
-                               @Value("${apiml.cloudGateway.cachePeriodSec:120}") int cachePeriodSec,
-                               @Value("${apiml.cloudGateway.clientKeystore:#{null}}") String clientKeystorePath,
-                               @Value("${apiml.cloudGateway.clientKeystorePassword:#{null}}") char[] clientKeystorePassword,
-                               @Value("${apiml.cloudGateway.clientKeystoreType:PKCS12}") String keystoreType
+    public GatewayIndexService(
+        @Qualifier("webClientClientCert") WebClient defaultWebClient,
```

Comment:
maybe change the name from default? it looks like this one is not default

Reply:
the word default removed